ui/src/app/shared/common.ts

```diff
@@ -7,7 +7,8 @@ export enum JobStatusIcon {
   Aborting = 'exclamation-triangle',
   Failed = 'times',
   Succeeded = 'check',
-  Aborted = 'exclamation-triangle'
+  Aborted = 'exclamation-triangle',
+  'On Hold' = 'pause'
```

Comment:
An identifier with a space in it looks really weird to me, but perhaps it's practical and/or conventional in Javascript-land

Reply:
Should the space get removed here as well?